docs/getting-started-guides/azure.md

```diff
@@ -0,0 +1,104 @@
+---
+---
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+1. An Azure Subscription. If you don't already have one, you may create one on [azure.microsoft.com](https://azure.microsoft.com).
+2. A Microsoft account with access to your own Subscription, or an organizational account with Owner access to the Subscription.
+3. Both `docker` and `jq` need to be installed and available on `$PATH`.
+
+
+## Cluster operations
+
+### Cluster bring-up
+
+```shell
+export KUBERNETES_PROVIDER=azure; curl -sS https://get.k8s.io | bash
+```
+
+NOTE: This script calls [cluster/kube-up.sh](http://releases.k8s.io/{{page.githubbranch}}/cluster/kube-up.sh)
+which in turn calls [cluster/azure/util.sh](http://releases.k8s.io/{{page.githubbranch}}/cluster/azure/util.sh)
+using [cluster/azure/config-default.sh](http://releases.k8s.io/{{page.githubbranch}}/cluster/azure/config-default.sh).
+
+You must set `AZURE_SUBSCRIPTION_ID` or you willl receive errors. It may be set in `cluster/azure/config-default.sh` or set as environment variables:
+
+```shell
+export AZURE_SUBSCRIPTION_ID="<your subscriptionID (guid)>"
+```
+
+These values can be overriden by setting them in `cluster/azure/config-default.sh` or as environment variables. They are shown here with their default values:
+
+```shell
+export AZURE_DEPLOY_ID="" # autogenerated if blank
+export AZURE_LOCATION="westus"
+export AZURE_RESOURCE_GROUP="" # generated from AZURE_DEPLOY_ID if unset
+export AZURE_MASTER_SIZE="Standard_A1"
+export AZURE_NODE_SIZE="Standard_A1"
+export AZURE_USERNAME="kube"
+export NUM_NODES=3
+export AZURE_AUTH_METHOD="device"
+```
+
+By default, this will deploy a cluster with 4 Standard\_A1-sized VMs: one master node, three worker nodes. This process takes about 5 to 10 minutes. Once the cluster is up, connection information to the cluster will be displayed. Additionally, your `kubectl` configuration will be updated to know about this cluster and this new cluster will be set as the active context.
```

Comment:
hmm maybe you can backtick the VM size as well for emphasis, that should eliminate the need for escaping.